test/src/e2e_vm_tests/test_programs/should_pass/language/const_global_shadow/json_abi_oracle.json

```diff
@@ -1,5 +1,4 @@
 {
-  "configurables": [],
```

Comment:
Why is `configurable` gone here?

Reply:
const_global_shadow test is disabled my guess  is that it wasn't updated in a while because it did throw any json ABI mismatch error previously.

Reply:
I probably had an json_abi_output.json in my pc from before it was disabled I will remove these changes.

Reply:
reverted these changes as they were from hanging json_abi_output.json obtained from before the tests were disabled.